### ISSUE TITLE
Deno error: Uncaught (in promise) ReferenceError: window is not defined. Replace `window` with `globalThis`.

### ISSUE THREAD

**Issue by amite** [qgustavor/mega#227]
**Describe the bug**
I am trying to run a simple script in a deno environment:

```sh
deno 2.1.1 (stable, release, x86_64-unknown-linux-gnu)
v8 13.0.245.12-rusty
typescript 5.6.2
```

```
error: Uncaught (in promise) ReferenceError: window is not defined
    at prepareKeyV2 (file:///home/amite/.cache/deno/npm/registry.npmjs.org/megajs/1.3.3/dist/main.browser-es.mjs:13941:3)
    at handleV2Account (file:///home/amite/.cache/deno/npm/registry.npmjs.org/megajs/1.3.3/dist/main.browser-es.mjs:16039:7)
    at file:///home/amite/.cache/deno/npm/registry.npmjs.org/megajs/1.3.3/dist/main.browser-es.mjs:16076:16
    at file:///home/amite/.cache/deno/npm/registry.npmjs.org/megajs/1.3.3/dist/main.browser-es.mjs:13747:27
    at eventLoopTick (ext:core/01_core.js:175:7)

    info: window global is not available in Deno 2.
    hint: Replace `window` with `globalThis`.

```

**To Reproduce**

Run this code in deno 2.1.1:

```js
import { Storage } from 'npm:megajs';

const email = "xxxx@gmail.com";
const password = "xxxxx";

// Function to recursively list files and folders
async function listFilesAndFolders(node: any, prefix = ''): Promise<string[]> {
    const children = await node.children; // Accessing children property
    const output: string[] = []; // Array to hold CSV formatted data
    for (const child of children) {
        output.push(`"${child.name}","${child.id}"`); // Collecting output in CSV format
    }
    return output;
}

// Function to find the "Tutorials" folder and list its contents
async function findAndListTutorialsFolder(storage: any) {
    const rootNode = await storage.root;
    const tutorialsFolder = await rootNode.find('Tutorials'); // Search for the Tutorials folder

    if (tutorialsFolder) {
        console.log(`Contents of the "Tutorials" folder:`);
        const contents = await listFilesAndFolders(tutorialsFolder); // List contents if found
        writeToCSV(contents); // Write contents to CSV file
    } else {
        console.log('The "Tutorials" folder was not found.');
    }
}

// Function to write contents to a CSV file
async function writeToCSV(contents: string[]) {
    const header = 'Name,ID\n'; // CSV header
    const csvData = header + contents.join('\n'); // Combine header with data

    try {
        await Deno.writeTextFile('contents.csv', csvData); // Writing to CSV file
        console.log('Contents written to contents.csv successfully.');
    } catch (err) {
        console.error('Error writing to file:', err);
    }
}

// Main execution block
(async function () {
    const storage = new Storage({
        email,
        password,
        autologin: true,
    });

    // Will resolve once the user is logged in or reject if some error happens
    await storage.ready;

    // Find and list the contents of the Tutorials folder
    await findAndListTutorialsFolder(storage);
})().catch(error => {
    console.error(error);
});
```

**Expected behavior**
Mega JS should list contents of the folder

**Additional context**
N/A

**Comment by qgustavor** [comment]
Should be fixed by 402d0404ed5749ffd7dd4b14bbb6f346f1db4ec2.

**Comment by qgustavor** [comment]
Can you check this? https://github.com/qgustavor/mega/pull/230
